<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePermissionGroup</class>
 <widget class="QDialog" name="CreatePermissionGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: #4D1F99;
}
QComboBox{
	background-color: #6F4AB1;
	color: #FFFFFF;
	font-weight: bold;
}
QPushButton {
	background-color: #6F4AB1;
	color: #FFFFFF;
	font-weight: bold;
    min-width: 80px;
}
QLineEdit{
	color: #000000;
	background-color: #FFFFFF;
}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="createpermissiongroup_lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="createpermissiongroup_listView">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="createpermissiongroup_rooms">
   <property name="geometry">
    <rect>
     <x>278</x>
     <y>130</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="createpermissiongroup_addroom">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createpermissiongroup_remove">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Group Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Person</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreatePermissionGroup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreatePermissionGroup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
